net/ldapvi/Portfile

```diff
@@ -25,11 +29,12 @@ pre-configure {
 }
 
 configure.cppflags-append "-L${prefix}/lib"
+configure.cflags-append -Wno-error=implicit-function-declaration
```

Comment:
instead of disabling the error ideally one should fix the implicit function declarations (this error will also occur on other systems than macOS 12). What is the error you get?

Reply:
port.c:100:22: error: implicit declaration of function 'yourfault' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
        if (onexitfunction) yourfault("on_exit called twice");
                            ^
ldapvi.c:580:2: error: implicit declaration of function 'on_exit' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
        on_exit((on_exit_function) cleanup, dir);
        ^
error.c:54:2: error: implicit declaration of function 'ldap_perror' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
        ldap_perror(ld, str);
        ^
sasl.c:72:2: error: implicit declaration of function 'fdcp' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
        fdcp(defaults->fd, 2);
        ^
port.c:132:2: error: implicit declaration of function 'RAND_bytes' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
        RAND_pseudo_bytes(rand, sizeof(rand));
        ^
diff.c:218:9: error: implicit declaration of function 'ldap_bv2dn_x' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
        ldap_bv2dn_x(&bv, out, flags);
        ^